lib/APINotes/APINotesYAMLCompiler.cpp

```diff
@@ -784,6 +786,8 @@ namespace {
         pInfo.SwiftName = prop.SwiftName;
         if (prop.Nullability)
           pInfo.setNullabilityAudited(*prop.Nullability);
+        if (prop.SwiftImportAsAccessors)
+          pInfo.setSwiftImportAsAccessors(*prop.SwiftImportAsAccessors);
```

Comment:
This file also has a translation in the *other* direction that you'll need to do, e.g., from the `ObjCPropertyInfo` to YAML. It's used to make sure everything round-trips through the binary format.